### DISCUSSION TITLE
Windows 10 v0.7.0

### DISCUSSION THREAD

**Discussion by ljramones** [ljramones/trips#337] [General]
lots of different items 

<hr /><em>This discussion was created from the release <a href='https://github.com/ljramones/trips/releases/tag/winv0.7.0'>Windows 10 v0.7.0</a>.</em>

**Comment by Cjburke23** [comment]
I tried to latest version on my ancient Windows 10 machine (it used to not be Win10).
My first try seemed to hang and I was about to download everything onto a laptop when I saw stuff appear in the command box. Then it loaded. 

I saw the note: Please import a dataset -> Star Data/Import data set from a file
Then Star Data/Select Retrieve Data Set

I never found those.
"Edit Star data" wasn't it.

I think Plot Stars is what got me to a list of files and I loaded one.  The Star labels are a big small for me to read. 

Logged out for dinner, came back a little later. It loaded a bit quicker. There wasn't as much lag in the Command box. My plot was still there. 
I hit Plot Stars, and it looked different, but I guess this was because now I had a dataset loaded. I loaded another file and it's been loading for about 10 minutes now.

The first one had 267 stars. This one has 212312. When I click on it, it says it's bad data, could not load. I don't know why.

Something is wrong with the small one now. I lot the blue lines. There's just a ring of dots. I click "grid" and the ring of dots goes away. There are no stars, just labels. Click on Stars and the labels go away. Ditto for hitting Star Labels. Extensions does nothing either. I guess I need to shut it off and reload.

Okay, enough for tonight.

**Reply by ljramones** [reply]
The label size for the stars is found by selected Preferences at the bottom of the second Menu
Select the Graph tab in preferences and the font as below


<img width="1048" alt="star font" src="https://user-images.githubusercontent.com/226595/152907902-a92373ab-8347-474a-8add-54ed20841563.png">

**Reply by ljramones** [reply]
I looked at the title of this item and I release that you are probably not using the correct version

V0.7.0 is an older version.

The correct version is v0.7.1 which is here: https://github.com/ljramones/trips/releases/tag/WinOS-v0.7.1

**Comment by ljramones** [comment]
Hi, The menu structure and order was chosen and designed by the community (I just work here).
Here are the steps to load data and plot it

1. First select Load dataset from this menu
<img width="379" alt="Import Dataset" src="https://user-images.githubusercontent.com/226595/152905357-7903c969-85d6-4637-9ddb-1023b2c97375.png">

2. The dataset manager should be visible
<img width="956" alt="Dataset Manager" src="https://user-images.githubusercontent.com/226595/152905503-af943d79-337f-468b-9754-072ad35c898a.png">

3. Select Add and give it a name
<img width="930" alt="Add Dataset" src="https://user-images.githubusercontent.com/226595/152905588-134509a4-93ab-418e-963e-84486180b343.png">

4. Select the csv file. You should already be in the directory of released data files (I selected the 270 ly one)
<img width="1274" alt="select the csv file" src="https://user-images.githubusercontent.com/226595/152905791-51ac34fe-1389-4551-a352-5373de7d96fd.png">

5. After you selected a file, push the add button. (For a 270 ly file, it takes a while to load)
<img width="1015" alt="select add" src="https://user-images.githubusercontent.com/226595/152905845-d15623b6-725f-4f25-b355-64bc31737a22.png">

6. You will start to see the file entries loading into the database. It loads 2000 entries at a time. It takes a while to load because of all the fields er entry
<img width="948" alt="loading of file" src="https://user-images.githubusercontent.com/226595/152906015-f6a8d9ff-a434-4dcd-8cec-edb0a40f5e00.png">

7. When the loading is complete, you'll see this
<img width="1018" alt="loading complete" src="https://user-images.githubusercontent.com/226595/152906182-ff9277cc-88ab-4015-8e11-7a3ed32c439b.png">

8. Now that the file is loaded into the DB, you can use the Plot Stars button
<img width="278" alt="plot stars" src="https://user-images.githubusercontent.com/226595/152906324-8dae8899-f2cf-433d-be19-072379d02a48.png">

9. When the stars are retrieved and plotted you see a plotting complete message (how many stars, etc)
<img width="1004" alt="plotting complete" src="https://user-images.githubusercontent.com/226595/152906393-2f43a256-f441-45a3-abd9-6fd9c9d04c83.png">

**Comment by ljramones** [comment]
There is a configuration item that allows you to change the label font size.

**Comment by ljramones** [comment]
for this one "The first one had 267 stars. This one has 212312. When I click on it, it says it's bad data, could not load. I don't know why."

the 270 ly file produces a database of 128 Meg. Its massive. If your computer isn't very powerful, it will take a while. For my laptop (8 core, 16 thread i9 with 32Gb of memory), it takes 1.5 minutes.

If it says bad data, you might have interrupted it before it could finish. To reset the database, go to the data directory and remove the file called tripsdb.mv.db. If you see tripsdb.trace.db, then remove that too. 
That file is the actual db. If the system doesn't see a file of this name, it creates an empty db file

**Comment by cegannon1** [comment]
Thinking that the order should be

import/load
open
save 1
save 2
export
quit

the first three follow the most common user sequence.

Didn't have time/distance to reflect upon and realize this when load was added to import.



Sent from my Verizon, Samsung Galaxy smartphone
Get Outlook for Android<https://aka.ms/AAb9ysg>
________________________________

**Reply by ljramones** [reply]
I will write up a ticket to change the order.

<img width="305" alt="CleanShot 2022-02-07 at 22 01 47@2x" src="https://user-images.githubusercontent.com/226595/152910051-50cc2abb-4d8b-4df9-86a7-089dcf34c3d2.png">

**Comment by cegannon1** [comment]
Hmmmm...is there  way to proof this against ~ user error. Given that a lit of machines aren't going to any more powerful than the one this user has, this could become a frequent fail path. And the reshuffling/deleting/renaming of files doesn't sound like most users would figure their way through it.

Sent from my Verizon, Samsung Galaxy smartphone
Get Outlook for Android<https://aka.ms/AAb9ysg>
________________________________

**Reply by ljramones** [reply]
The 270 ly file is massive. The resulting DB is just over 128 Mb.
I also think he was running the wrong version. He posted the issue and related it to v0.7.0
If he cancels in the middle of a load then the load result is undefined for that one. I could force a purge of the partial load. 
However, he could delete the bad data and reload again.
And he said his machine was ancient. Ancient can mean a lot of things. THere are real limits based on the size of the data you try to load. It will work if you are patient and don't try to abort in the middle of it.

**Comment by cegannon1** [comment]
Great! Thanks!

Sent from my Verizon, Samsung Galaxy smartphone
Get Outlook for Android<https://aka.ms/AAb9ysg>
________________________________
